src/datagrid/DataGrid.stories.tsx

```diff
@@ -24,6 +24,7 @@ import {
     paginationDetails,
     paginationRows,
     pageFilterFunction,
+    hideNshowCol,
```

Comment:
Can we rename hideNshowCol and/or hideShowCol to hideableColumn in all the references ?

Reply:
done